client/src/containers/VerifyEmailPage.tsx

```diff
@@ -1,44 +1,148 @@
+
+  useEffect(() => {
+    const asyncFetchUser = async () => {
+      const _user = await AuthClient.fetchUser();
+      if (_user) {
+        setUser({
+          ..._user,
+          subscriptions:
+            _user.subscriptions?.map((s: any) => {
+              return { ...s };
+            }) || [],
+        });
+        setIsVerified(_user.verified);
+      }
+    };
+    asyncFetchUser();
+  }, []);
+
+  const delaySeconds = 5;
+  const baseUrl = window.location.origin;
+  const redirectUrl = `${baseUrl}/${i18n.language}/account/settings`;
+
+  const updateCountdown = () => {
+    let timeLeft = delaySeconds;
+    const delayInterval = delaySeconds * 100;
+
+    setInterval(() => {
+      timeLeft--;
+      document.getElementById("countdown")!.textContent = timeLeft.toString();
+      if (timeLeft <= 0) {
+        document.location.href = redirectUrl;
+      }
+    }, delayInterval);
+  };
+
+  const renderLandingPage = () => {
+    const expiredLinkPage = (
+      <Fragment>
+        <br />
+        <Trans render="h3" className="text-center">
+          The link sent to you has timed out.
+        </Trans>
+        <br />
+        <div className="text-center">
+          <button
+            className="button is-secondary"
+            onClick={async () => {
+              setShowConfirmation(await AuthClient.resendVerifyEmail());
+            }}
+          >
+            <Trans>Resend verification email</Trans>
+          </button>
+        </div>
+        <br />
+      </Fragment>
+    );
+
+    const validLinkPage = (
+      <Fragment>
+        <Trans render="h3"> Verify this email: </Trans>
+        <br />
+        <h3 className="text-center">{!!user && user.email}</h3>
+        <br />
+        <Trans render="h3"> to receive Data Updates from Who Owns What. </Trans>
+        <div className="text-center">
+          <button
+            className="button is-primary"
+            onClick={() => {
+              asyncVerifyEmail().then((result) => {
+                const isVerified = result.statusCode === VerifyStatusCode.Success;
+                setIsVerified(isVerified);
+                setIsLinkExpired(result.statusCode === VerifyStatusCode.Expired);
+                setShowConfirmation(isVerified);
+              });
+            }}
+          >
+            <Trans>Verify email</Trans>
+          </button>
+        </div>
+      </Fragment>
+    );
+    return isLinkExpired ? expiredLinkPage : validLinkPage;
+  };
+
+  const asyncVerifyEmail = async () => {
+    return await AuthClient.verifyEmail();
+  };
+
+  const renderConfirmationPage = () => {
+    const resendEmailPage = (
+      <Fragment>
+        <Trans render="h3" className="text-center">
+          Check your email inbox & spam
+        </Trans>
+        <br />
+        <Trans className="text-center">
+          Click the link we sent to verify your email address {!!user && user.email}. It may take a
```

Comment:
I think this is just a lingui thing, but in testing this I got `{0}` for the email
![image](https://github.com/JustFixNYC/who-owns-what/assets/16906516/41a3c054-e8ae-404f-aea9-2b1c948e9520)